R/database.R

```diff
@@ -1505,23 +1504,24 @@ get_ssm_by_region = function(chromosome,
                              from_indexed_flatfile = TRUE,
                              augmented = TRUE,
                              min_read_support = 3,
-                             mode = "slms-3",
-                             allow_clustered = TRUE){
+                             mode = "slms-3"){
 
   tabix_bin = "/home/rmorin/miniconda3/bin/tabix"
   table_name = config::get("results_tables")$ssm
   db = config::get("database_name")
+
   if(from_indexed_flatfile){
     base_path = config::get("project_base")
 
     #test if we have permissions for the full gambl + icgc merge
     if(mode == "slms-3"){
-      if(allow_clustered && !augmented){
+      if(!augmented){
         maf_partial_path = config::get("results_flatfiles")$ssm$template$merged$deblacklisted
       }
-      if(allow_clustered && augmented){
+      if(augmented){
         maf_partial_path = config::get("results_flatfiles")$ssm$template$merged$augmented
       }
```

Comment:
Why isn't this the more logical:

if(augmented){
maf_partial_path = config::get("results_flatfiles")$ssm$template$merged$augmented
}else{
maf_partial_path = config::get("results_flatfiles")$ssm$template$merged$deblacklisted
}